### ISSUE TITLE
CH4 Sounds

### ISSUE THREAD

**Issue by gap9269** [gap9269/SDDSound#362]
Ambience (only listed ones for old maps or ones that need a blend)
----

these can be 'tear ambiences' that you just place in the map, then use a normal ambience as a bed for the rest of it

X ambience_camp_inferno (Napoleon's CH4 camp)
      - ambience_camp_inferno_blend (Blend for the hell hole in the center)
X ambience_lava_field
      - ambience_lava_field_blend (for the middle lava part)
X ambience_princess_room_2 (the lab stuff)

Princess Room Stuff:
---
X princess_lever (when she pulls the lever in her room)
X princess_belt_start
X princess_belt_stop
X princess_portal_power_on
X princess_portal_activate

Science:
----
X save_scientist (a little noise for when the red bubble fades out)
- Glitched ambiences and sounds (Add what you think is necessary for the backwards science maps)
X karin_time_travel_defeat (in the cutscene when daryl time travels to beat Karin)

History:
---
X small_tear_loop_start (the tiny tears into the new map branches. Plus the one behind the great wall we never did)
X history_flag_change

Inferno:
---
X cutscene_minotaur_tim_transform (same as gorilla tim, but needs to change for when the ground breaks and he falls)

Misc:
----
X map_glitch (plays each time a map glitches out. Not the constant ones, but the more significant, spaced out glitches. It's used a lot so maybe just put this in the global bank)

X cutscene_locker_alarm (For when you break into Paul's locker)
X cutscene_locker_alarm_stop

X cutscene_party_noise (classic party confetti farty noise in the janitor's closet scene when confetti falls)

Move to loose objects (keep in their normal banks tho)
---
X object_block_crumble
X frankenstein_gate_raise

Weasel Quest:
---
X weasel_death (starts with the hand appearing)

Watson Quest
---
- watson_science_portal_activate (when the enemies stop spawning in the Axis and the Tear is usable. coincides with the screen shaking. Just some sort of noise to let you know you did something.) **Note: We might already have this with the new portal sounds that you made**

**Comment by ghost** [comment]
~Use Music_Playback_Speed RTPC to set music to 86% in tear maps~

**Comment by ghost** [comment]
Add computer Science 103 platform stuff to loose objects, use in city of ghosts for green plats

use laser fence on/off too instead of door lock sound

**Comment by ghost** [comment]
Sire Cymbal:
- ramble loop
- giant cymbal slam

**Comment by ghost** [comment]
Don pecker collapse

**Comment by gap9269** [comment]
Low Priority:
---
- ambience_intro_to_history_altered (CH4 version)
- Shattered_Fort_Post_Bomb (Thought it would be cool for the music and ambience to fade out as you reach the top, but only after the bomb has gone off. So I added this RTPC)
X object_burning_locker
X object_burning_booth
X tuba_ghost_prison_start (new paranormal room. Just needs to sound quiet like he's in the prison room)
X tuba_ghost_prison_stop
X open_weasel_cage (for history weasel quest)
X weasel_run
- history_melting_wall_disappear (in "Not a Battlefield" when you press the button)

**Comment by gap9269** [comment]
- cutscene_minotaur_tim_transform : This could use a falling and crashing sound for the final version

**Comment by gap9269** [comment]
- Added "patroll_summon" to the patroll. Same as "troll_summon"
